addons/web_editor/static/lib/odoo-editor/test/spec/list.test.js

```diff
@@ -4140,16 +4140,7 @@ describe('List', () => {
                                     </ul>`),
                                 stepFunction: deleteBackward,
                                 contentAfter: unformat(`
-                                    <ul class="o_checklist">
-                                        <li>abc</li>
-                                        <li>[]<br></li>
-                                        <li class="oe-nested">
-                                            <ul class="o_checklist">
-                                                <li><br></li>
-                                            </ul>
-                                        </li>
-                                        <li class="o_checked">def</li>
-                                    </ul>`),
+                                <ul class="o_checklist"><li>abc</li></ul><p>[]<br></p><ul class="o_checklist"><li class="oe-nested"><ul class="o_checklist"><li><br></li></ul></li><li class="o_checked">def</li></ul>`),
```

Comment:
Please keep the html properly formatted here, otherwise it is very hard to read.